.github/ISSUE_TEMPLATE/feature-request---proposal.md

```diff
@@ -16,15 +16,15 @@ A clear and concise description of what you want to happen.
 **Is this a breaking change**
 A breaking change would require consumers or implementors of the API to modify their code for it to continue to function (ex: renaming of a required field or the change in data type of an existing field). A non-breaking change would allow existing code to continue to function (ex: addition of an optional field or the creation of a new optional endpoint). 
 
-[  ] Yes, breaking
-[  ] No, not breaking
-[  ] I'm not sure
+* [  ] Yes, breaking
+* [  ] No, not breaking
+* [  ] I'm not sure
```

Comment:
You may want to also only put one space inside the brackets so they render as checkboxes 🤷‍♂️